### PR TITLE
Add PrivateIPAddressVersion in LoadBalancerFrontendIpConfiguration.

### PR DESCRIPTION
**Pull request by jaepark08**
## Description

Add PrivateIPAddressVersion property in LoadBalancerFrontendIpConfiguration.

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by jaepark08** [comment]
https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/319#issuecomment-501367076

File: src/Network/Network/Generated/LoadBalancer/FrontendIPConfiguration/AddAzureRmLoadBalancerFrontendIpConfigCommand.cs [hunk 1]
```diff
@@ -61,6 +61,22 @@ public partial class AddAzureRmLoadBalancerFrontendIpConfigCommand : NetworkBase
...
+            ParameterSetName = "SetByResourceSubnet",
+            HelpMessage = "The private IP address version of the IP configuration.",
+            ValueFromPipelineByPropertyName = true)]
+        [Parameter(
+            Mandatory = false,
+            ParameterSetName = "SetByResourceIdSubnet",
+            HelpMessage = "The private IP address version of the IP configuration.",
+            ValueFromPipelineByPropertyName = true)]
+        [ValidateSet(
+            MNM.IPVersion.IPv4,
```
Comment (number213) [inline src/Network/Network/Generated/LoadBalancer/FrontendIPConfiguration/AddAzureRmLoadBalancerFrontendIpConfigCommand.cs]: Missing import (`using MNM = Microsoft.Azure.Management.Network.Models;`) is breaking build. Same thing in `New` and `Set`

**Comment by jaepark08** [comment]
> Please add test coverage for new feature, re-generate help files and rebase to `network-may`

Done.

**Comment by number213** [comment]
Tried running tests and found an issue with SDK. Swagger for this feature has been [merged](https://github.com/Azure/azure-rest-api-specs/pull/6162) after SDK has been [released](https://github.com/Azure/azure-sdk-for-net/pull/6407). We'll need to wait for the next release, I guess.
@MikhailTryakhov should this be retargeted to `network-may`?

**Comment by cormacpayne** [comment]
@jaepark08 @number213 I'm going to temporarily close this PR due to lack of activity. Feel free to re-open this PR whenever all of the necessary changes are in the SDK and pushed into this PR.

**Pull request closed without merging**